tests/clpy_tests/opencl_tests/test_elementwise.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import numpy
+
+import clpy
+from clpy import testing
+
+
+@testing.gpu
+class TestUltima(unittest.TestCase):
+
+    def test_implicit_conversion_with_constructor(self):
+        x = clpy.core.array(numpy.array([1]))
+        clpy.ElementwiseKernel(
```

Comment:
Why does not this test check any result? What is expected?

How about following?

1. x = clpy.array([1])
2. struct has `v` as member
3. self.v = x + 1 at ctor
4. test t = 3
5. y = t.v
6. expected y = 4

Reply:
This test can check that **ultima supports implicit "conversion with constructor" syntax as input** .

However, the test doesn't check any semantics. I'll fix it.